Improved 'get data ids' of xcube-cmems datastore (#19)
dcs4cop/xcube-cmems: CHANGES.md

## Changes in 0.1.2

> - Improved the performance of get_data_ids method of the store by using asyncio and caching,closes #18

Reviewer: ```suggestion
- Improved the performance of get_data_ids method of the store by using asyncio and caching. (#18)

```

## Changes in 0.1.1

- Fixed a bug when opening dataset with bounding box. Now the following spatial coords naming styles are
  supported: (lat,lon), (latitude,longitude), (y,x).

> - set cookie 'CASTGC' optionally to avoid users from getting KeyError, closes #12

Reviewer: ```suggestion
- Set cookie 'CASTGC' only if it previously existed to avoid users from getting KeyError. (#12)

```

> - addressed inconsisent parameter names for credentials and rely on global environment variables
>   CMEMS_USERNAME and CMEMS_PASSWORD, closes  #11, #15 , #16, #17

Reviewer: ```suggestion
- Addressed inconsistent parameter names for credentials and rely on global environment variables
  CMEMS_USERNAME and CMEMS_PASSWORD.  (#11) (#15) (#16) (#17)

```
